src/backend/web/routes/login.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const passport = require('passport');
+const { samlStrategy, handleLogin } = require('../../login/usingPassport');
+const logger = require('../../utils/logger');
+
+const router = express.Router();
+
+router.use(cookieParser());
+
+router.use(bodyParser.urlencoded({ extended: false }));
+router.use(bodyParser.json());
+
+const { SAML2_CLIENT_SECRET } = process.env;
+router.use(session({ secret: SAML2_CLIENT_SECRET, resave: false, saveUninitialized: true }));
```

Comment:
For all these variables you need from the `env`, should we log if they are missing?  What will happen if we don't have them set in a dev environment and you run the server?

Reply:
Everything I believe will still run, the login should break though if they aren't filled out, however it shouldn't stop someone from being able to add in code if they were working on redis or other parts of the website (at least currently.)